packages/account/src/Components/poi/poi-form-on-signup/idv-doc-submit-on-signup/idv-doc-submit-on-signup.tsx

```diff
@@ -16,15 +16,12 @@ import FormSubHeader from 'Components/form-sub-header';
 import IDVForm from 'Components/forms/idv-form';
 import PersonalDetailsForm from 'Components/forms/personal-details-form';
 import FormFooter from 'Components/form-footer';
-import { GetSettings } from '@deriv/api-types';
+import type { GetSettings } from '@deriv/api-types';
 
 type TIdvDocSubmitOnSignup = {
     citizen_data: FormikValues;
     onPrevious: (values: FormikValues) => void;
-    onNext: (
-        values: FormikValues,
-        action: FormikHelpers<{ document_type: FormikValues; document_number: FormikValues }>
-    ) => void;
+    onNext: (values: FormikValues, action: FormikHelpers<FormikValues>) => void;
```

Comment:
Can type `TFormProps` be put under types and used over here instead of FormiValues? The type is already exported out in this PR https://github.com/binary-com/deriv-app/pull/9618, if u want to refer.

Reply:
Let’s fix it throughout the package in separate PR, how do you think?